requirements.txt

```diff
@@ -24,6 +24,7 @@ django-mptt==0.11.0             # Modified Preorder Tree Traversal
 django-redis>=5.0.0
 django-q==1.3.4                 # Background task scheduling
 django-sql-utils==0.5.0         # Advanced query annotation / aggregation
+django-sslserver==0.22          # Secure HTTP development server
```

Comment:
Why is this package added?

Reply:
Pages have to be served over https in order to access the camera, so I needed a simple way to spin up a development server.